main.js

```diff
@@ -0,0 +1,169 @@
+import {
+  getProductList,
+  getProduct,
+  createProduct,
+  patchProduct,
+  deleteProduct,
+} from "./ProductService.js";
+import { PRODUCT, ElectronicProduct } from "./ProductClass.js";
+import {
+  getArticleList,
+  getArticle,
+  createArticle,
+  patchArticle,
+  deleteArticle,
+} from "./ArticleService.js";
+import { ARTICLE } from "./ArticleClass.js";
+
+function addArtget() {
+  let thing;
+  thing = new ARTICLE(arti.title, arti.content);
+
+  return thing;
+}
```

Comment:
`arti` 변수가 함수 내부나 외부에서 정의되지 않았습니다. 
139번째 줄 주석을 보면 `getArticle`로부터 받으려고 했던 것 같은데, 현재 이 함수는 제대로 동작하지 않을 거예요.
아래와 같이 수정이 필요해보입니다.
```suggestion
function addArtget(articleData) {
  return new ARTICLE(articleData.title, articleData.content);
}
```